pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -0,0 +1,289 @@
+import warnings
```

Comment:
If it makes sense, I think a separate `test_cov.py` and `test_corr.py` would be good.